presto-docs/src/main/sphinx/installation.rst

```diff
@@ -2,11 +2,18 @@
 Installation
 ************
 
+A Presto server can be installed and deployed on various systems. Typically you
+run a cluster of machines with one coordinator and many workers. You can find
+instructions for deploying such a cluster and related information in the
+following sections:
+
 .. toctree::
     :maxdepth: 1
 
     installation/deployment
-    installation/cli
-    installation/jdbc
```

Comment:
After this change i will no longer be able to:

1. open https://prestosql.io/docs/current/
2. search for "JDBC"

to find docs for the JDBC driver.

I am against removing top-level items from docs.

Reply:
yes you will .. it will show up in the clients section 

<img width="463" alt="Screen Shot 2020-06-12 at 09 25 45" src="https://user-images.githubusercontent.com/145658/84524416-cd6c9400-ac8e-11ea-9035-fb44fc38d45d.png">

Reply:
Thanks!